my-index.ejs

```diff
@@ -9,8 +9,8 @@
         // Set default API in case none is specified in .env
         __API_ROOT__ = 'cbioportal-rc.herokuapp.com/api';
     </script>
-    <script src="reactapp/js/common.bundle.js"></script>
-    <link href="reactapp/css/bootstrap.min.css" rel="stylesheet"></link>
+    <script src="reactapp/common.bundle.js"></script>
+    <link href="reactapp/prefixed-bootstrap.min.css" rel="stylesheet"></link>
```

Comment:
Is it essential to change these paths? Might be better to keep them the same, so in the parent cBioPortal project we don't have to change anything.